docs/RationaleRTL-SV.md

```diff
@@ -53,6 +53,59 @@ TODO: Describe inout types.  Analogy to lvalues vs rvalues.  Array indices for
 both forms.  Arrays, structs,
 moving [UnpackedArray](https://github.com/llvm/circt/issues/389) to SV someday.
 
+All types (excepting `UnpackedArray`) are packed, meaning no padding or
+alignment.
+
+### Integrals
```

Comment:
I'd recommend naming this section "Integer bit vectors", and "integer" instead of "integral" in the description.

Maybe it is just me but the word integral means the calculus thing.

Reply:
The SystemVerilog spec uses the term "integral" but I agree with you.